src/main/java/com/example/demoadminpanel/user/controller/UserController.java

```diff
@@ -27,41 +28,42 @@ public class UserController {
     @GetMapping()
     @PreAuthorize("hasRole('ADMIN')")
     @ResponseStatus(HttpStatus.OK)
-    public List<UserListResponse> getUsers() {
-        return userService.getUsers();
+    public ResponseEntity<List<UserListResponse>> getUsers() {
```

Comment:
Next time refactor from list to page when fetching content (will need frontend modifications too)